config/main.py

```diff
@@ -5429,6 +5429,83 @@ def disable_use_link_local_only(ctx, interface_name):
     interface_dict = db.get_table(interface_type)
     set_ipv6_link_local_only_on_interface(db, interface_dict, interface_type, interface_name, "disable")
 
+#
+# 'tx_error_threshold' subgroup ('config tx_error_threshold <interface_name> ...')
+#
+
+@interface.group('tx_error_threshold')
+@click.pass_context
+def tx_error_threshold(ctx):
+    """Set or del threshold of tx error statistics"""
+    pass
+
+#
+# 'set_thresh' subcommand
+#
+
+@tx_error_threshold.command()
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('interface_tx_err_threshold', metavar='<interface_tx_err_threshold>', required=True, type=int)
+@click.pass_context
+def set_thresh(ctx, interface_name, interface_tx_err_threshold):
+    config_db = ctx.obj['config_db']
+
+    """Set threshold of tx error statistics"""
+    if interface_name is None:
+        ctx.fail("'interface_name' is None!")
+
+    if clicommon.get_interface_naming_mode() == "alias":
+        interface_name = interface_alias_to_name(interface_name)
+        if interface_name is None:
+            ctx.fail("'interface_name' is None!")
+
+    if interface_name_is_valid(config_db,interface_name) is False:
+        ctx.fail("Interface name is invalid. Please enter a valid interface name!!")
+
+    if interface_name.startswith("Ethernet"):
+        config_db.set_entry("TX_ERR_CFG", (interface_name), {"port_tx_error_threshold": interface_tx_err_threshold})
+    else:
+        ctx.fail("Only Ethernet interfaces are supported")
+
+#
+# 'clear_thresh' subcommand
+#
+        
+@tx_error_threshold.command()
+@click.pass_context
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+def clear_thresh(ctx, interface_name):
```

Comment:
The only difference between the `set_thresh` and `clear_thresh` is that in that `clear_thresh` uses None instead of the actual threshold value. Please remove the code duplication.